BUILDING.md

```diff
@@ -61,7 +72,7 @@ Once the Python binary wheel file (`ngraph-*.whl`) is prepared you can install i
 
 For example:
 
-    (your_venv) $ pip install -U dist/ngraph-0.10.0-cp35-cp35m-linux_x86_64.whl
+    (nGraph) $ pip install -U dist/ngraph_core-0.0.0.dev0-cp35-cp35m-linux_x86_64.whl
```

Comment:
If we build wheel won't we have a specific package version in wheel name?

Reply:
Not if we use `--single-branch --depth 1 `